security-proxy/src/main/java/org/georchestra/security/Proxy.java

```diff
@@ -142,6 +142,7 @@
      */
     private String defaultTarget;
     private String publicHostname = "https://georchestra.mydomain.org/";
```

Comment:
IIRC `publicHostname` should not include a final slash, see eg https://github.com/georchestra/datadir/blob/509185f550cab4ebc098baf76fc5a5a1d8ae53d3/default.properties#L3-L5